ui/ui.go

```
@@ -72,3 +80,35 @@ func healthcheckHandler(cfg Config) http.Handler {
 		fmt.Fprintf(w, "OK")
 	})
 }
+
+func debugHandler(cfg Config) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+
+		cfg.DB.DbUser = "********"
+		cfg.DB.DbPassword = "********"
+		cfg.Publicip = publicIP()
+		cfg.Hostname, _ = os.Hostname()
+		cfg.Timestamp = time.Now().Unix()
+		bytes, err := json.Marshal(cfg)
+
+		if err != nil {
+			fmt.Fprintf(w, "{\"error\": \"JSON Marshal error\"}")
+			return
+		}
+		fmt.Fprintf(w, string(bytes))
+	})
+}
+
+func publicIP() string {
+	url := "https://checkip.amazonaws.com/"
+	resp, err := http.Get(url)
+	if err != nil {
+		panic(err)
```

Why panic() instead of returning a string like "unavailable" or "unknown" ?